hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -2942,6 +2950,22 @@ public int getSecondaryIndexParallelism() {
     return metadataConfig.getSecondaryIndexParallelism();
   }
 
+  /**
+   * Whether to enable Optimized writes or not. By default for table version 6 we are not enabling this, since NBCC is not available in table version 6.
+   * For flink and java engines, there are no issues w/ task retries and so they are out of it.
```

Comment:
`they are out of it` is not clear. Is it simply `flink and java do not have a need for this feature since the do not have task retries`

Reply:
sg.